File: java/ProgramATM.java
import java.util.Scanner;

public class ProgramATM {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        String nama, benar;

        while (true) {
            System.out.print("Masukkan nama: ");
            nama = input.next();

            System.out.print("Nama benar? (true/false): ");
            benar = input.next();

            if (benar.equals("true")) break;
        }

        System.out.print("Masukkan NIM (angka saja): ");
        int nim = input.nextInt();

        int saldo = nim;

        System.out.println("====================");
        System.out.println("Nama: " + nama);
        System.out.println("Saldo awal: Rp " + saldo);

        System.out.println("===== MENU ATM =====");
        System.out.println("1. Cek Saldo");
        System.out.println("2. Tarik Tunai");
        System.out.println("3. Setor Tunai");
        System.out.println("4. Transfer");
        System.out.println("5. Keluar");
        System.out.println("====================");

        System.out.print("Pilih menu: ");
        int menu = input.nextInt();

        switch (menu) {
            case 1:
                System.out.println("Saldo: " + saldo);
                break;

            case 2:
                System.out.print("Jumlah tarik: ");
                int tarik = input.nextInt();
                System.out.println("Saldo sekarang: " + (saldo - tarik));
                break;

            case 3:
                System.out.print("Jumlah setor: ");
                int setor = input.nextInt();
                System.out.println("Saldo sekarang: " + (saldo + setor));
                break;

            case 4:
                System.out.print("Jumlah transfer: ");
                int tf = input.nextInt();
                System.out.println("Saldo sekarang: " + (saldo - tf));
                break;

            case 5:
                System.out.println("Keluar");
                break;

            default:
                System.out.println("Menu tidak valid");
        }

        input.close();
    }
}

File: c++/Program-ATM.cpp
#include <iostream>
using namespace std;

int main() {
    string nama, benar;

    while (true) {
        system("cls");
        cout << "Masukkan nama: ";
        cin >> nama;
        cout << "Nama benar? (true/false): ";
        cin >> benar;

        if (benar == "true") break;
    }

    int nim;
    cout << "Masukkan NIM (angka saja): ";
    cin >> nim;

    int saldo = nim;

    cout << "====================" << endl;
    cout << "Nama: " << nama << endl;
    cout << "Saldo awal: Rp " << saldo << endl;

    cout << "===== MENU ATM =====" << endl;
    cout << "1. Cek Saldo" << endl;
    cout << "2. Tarik Tunai" << endl;
    cout << "3. Setor Tunai" << endl;
    cout << "4. Transfer" << endl;
    cout << "5. Keluar" << endl;
    cout << "====================" << endl;

    int menu;
    cout << "Pilih menu: ";
    cin >> menu;

    switch (menu) {
        case 1:
            cout << "Saldo: " << saldo << endl;
            break;

        case 2: {
            int tarik;
            cout << "Jumlah tarik: ";
            cin >> tarik;
            cout << "Saldo sekarang: " << saldo - tarik << endl;
            break;
        }

        case 3: {
            int setor;
            cout << "Jumlah setor: ";
            cin >> setor;
            cout << "Saldo sekarang: " << saldo + setor << endl;
            break;
        }

        case 4: {
            int tf;
            cout << "Jumlah transfer: ";
            cin >> tf;
            cout << "Saldo sekarang: " << saldo - tf << endl;
            break;
        }

        case 5:
            cout << "Keluar" << endl;
            break;

        default:
            cout << "Menu tidak valid" << endl;
    }

    return 0;
}
